src/util/PopulationsMap.test.ts

```diff
@@ -10,7 +10,7 @@ import {
 import {
   triggerPopChanges,
   getValueFromBoolOrNum,
-  mapExistingTestCase,
+  mapExistingTestCasePopulations,
 } from "../util/PopulationsMap";
```

Comment:
test file is from the same directory, we could do 
```suggestion
} from "./PopulationsMap";
```

Reply:
I did direct import from vs code suggestion, so i haven't checked it much. I'll update it